components/seo/head-seo.tsx

```diff
@@ -66,11 +67,15 @@ const buildSeoMeta = (pageProps: {
   };
 };
 
-const constructImage = (name: string, avatar: string, description: string): string => {
+const constructImage = (name: string, description: string, username: string): string => {
   return (
     encodeURIComponent("Meet **" + name + "** <br>" + description).replace(/'/g, "%27") +
```

Comment:
i18n consideration here.

Reply:
nope because the location is not clear when generating the social preview so we go with english only